Intersect.Client.Core/Entities/Player.cs

```diff
@@ -1960,6 +1961,11 @@ public bool TryAttack()
             return false;
         }
 
+        if (DragAndDrop.IsDragging)
+        {
+            return false;
+        }
```

Comment:
This does not belong here.

Presumably this is to stop the left click from triggering an attack when dragging and dropping, but this should be processed in the controls instead (i.e. if dragging and dropping, the controls binding should not return "active" for left mouse button). There shouldn't be controls logic in this method.